templates/selenium-python37/src/__ALGO__.py

```diff
@@ -0,0 +1,31 @@
+import Algorithmia
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+
+def get_chrome():
+    chrome_options = webdriver.ChromeOptions()
+    chrome_options.add_argument('--no-sandbox')
+    chrome_options.add_argument('--window-size=1420,1080')
+    chrome_options.add_argument('--headless')
+    chrome_options.add_argument('--disable-gpu')
+    driver = webdriver.Chrome(chrome_options=chrome_options)
+    return driver
+
+def get_phantomjs():
+    driver = webdriver.PhantomJS()
+    return driver
+
+# API calls will begin at the apply() method, with the request body passed as 'input'
+# For more details, see algorithmia.com/developers/algorithm-development/languages
+def apply(input):
+    driver = get_chrome()
+    #driver = get_phantomjs()
```

Comment:
is this an alternative? Can we document it instead of just having it like this. or remove it